uavcan/driver.py

```diff
@@ -1,13 +1,15 @@
 #encoding=utf-8
 
 import os
+import sys
 import time
 import socket
+import fcntl
 import struct
 import binascii
 import functools
 import logging as log
-
+from _socketmodule import SOL_SOCKET
```

Comment:
This should be moved in the import exception handler below, right under `except Exception:`.